### PR TITLE
Moving VSCode from `TRIAL` to `ADOPT`

### PR DESCRIPTION
**Pull request by DeepakThi**
I'm suggesting that we move VSCode from `TRIAL` to `ADOPT`.

**Reason** - Visual Studio Code is an opensource, lightweight, cross-platform editor for most programming languages & files, with amazing extensions and built-in space for terminals.

(also to see how the PR discussion would go for a change like this..  :D)

### CODE REVIEW

**Review by igorm-qube — APPROVED** [review]

File: Qube TechRadar.csv [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name,ring,quadrant,isNew,description
 Go,Adopt,languages-and-frameworks,FALSE,"Go is an open source programming language that makes it easy to build simple, reliable, and efficient software."
-Visual Studio Code,Trial,Tools,FALSE,"Visual Studio Code is a code editor redefined and optimized for building and debugging modern web and cloud applications."
+Visual Studio Code,Adopt,Tools,TRUE,"Visual Studio Code is an opensource, lightweight, cross-platform code editor for most programming languages & files, with amazing extensions and built-in space for terminals"
```
Comment (nileshr) [inline Qube TechRadar.csv]:
```suggestion
Visual Studio Code,Adopt,Tools,TRUE,"Visual Studio Code is an open-source, cross-platform code editor for most programming languages & files, with amazing extensions and built-in space for terminals"
```

Comment (nileshr) [reply Qube TechRadar.csv]: Although it has vastly improved, I think better not to say lightweight.

Comment (DeepakThi) [reply Qube TechRadar.csv]: Okay, I can live with that..  :D

**Review by nileshr — APPROVED** [review]

**Review by sivachandran — APPROVED** [review]

**Review by grkkarthick90 — APPROVED** [review]

**Comment by DeepakThi** [comment]
Not much discussion on this so far.  @Iyer-VivekAnand, your thoughts?

**Comment by Iyer-VivekAnand** [comment]
> Not much discussion on this so far. @Iyer-VivekAnand, your thoughts?

Agree @DeepakThi .

**Review by Iyer-VivekAnand — APPROVED** [review]

**Pull request merged**